src/modules/robot/Robot.cpp

```diff
@@ -356,6 +362,7 @@ void Robot::on_gcode_received(void *argument)
                 }
                 gcode->mark_as_taken();
             }
+            break; //added by DHP to suppress warning 20150324
```

Comment:
this is actually a good catch and is a bug. Not sure why the tool chain does not return this as an error